07-Patchy-Particle-Simulations-In-HPMC/02-Triblock-Janus-particle-simulation.ipynb

```diff
@@ -0,0 +1,601 @@
+{
```

Comment:
<p>HOOMD implicitly assumes $\beta = 1$ only for HPMC? </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Yes, good catch. I fixed that line in the notebook.